hooks/pre-commit

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# pre-commit hook for git which builds the docker image for a given cloud-sdk version.
+# Copy into .git/hooks/  folder
+# edit alpine/Dockerfile and  set CLOUD_SDK_VERSION, SHA256SUM
+# then export TAG_VER=153.0.0
```

Comment:
tag_ver is uses later on in the flow while actually pushing the tag (see maintenence.md)